### DISCUSSION TITLE
Screenshot not attaching to the gauge report

### DISCUSSION THREAD

**Discussion by ravikiranhebbar88** [getgauge/taiko#2202] [General]
In gauge after running the scenario captures the screenshot and doesn't attach to the report. Below code is used in a hook to capture the screenshot.

`gauge.customScreenshotWriter = async function () {
    const screenshotFilePath = path.join(process.env['gauge_screenshots_dir'], `screenshot-${process.hrtime.bigint()}.png`);
    await screenshot({ path: screenshotFilePath });
    return path.basename(screenshotFilePath);
};

afterScenario(async function(context) { 
    if(context.currentScenario.isFailed) {
       gauge.customScreenshotWriter();
    }
    await closeBrowser();
});
`
Do I need to add any code to attach to the report or does it add automatically? 

Can anyone help me out resolving this issue. Thanks in Advance

**Comment by zabil** [comment]
You need to call `gauge.screenshot()` to take screenshots. More details at
https://docs.gauge.org/writing-specifications.html?os=macos&language=javascript&ide=vscode#adding-screenshots-in-reports

**Reply by ravikiranhebbar88** [reply]
@zabil I had tried with `gauge.screenshot() ` in the after scenario hook and it is taking the screenshot, but not attaching it in the report.

**Reply by ravikiranhebbar88** [reply]
@zabil now, it is working after reinstalling the gauge :)

![image](https://user-images.githubusercontent.com/12529893/126043723-32f1a511-735d-44ce-be1c-050012ccd27c.png)
